.github/workflows/windows_workflow.yml

```diff
@@ -0,0 +1,35 @@
+name: windows-workflow
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: windows-latest
```

Comment:
Is there a set windows version? Do we know it? 
Do they update it often?
Windows is notoriously unstable across versions so I want to be sure we are constant.

Reply:
Version is specified at https://github.com/actions/virtual-environments

GitHub team make a version as `-latest` only when it is stable (ex. ubuntu still use 18.04 as latest)

Reply:
👍